### PR TITLE
INT-2542 update presentation in operator catalog

### PR DESCRIPTION
**Pull request by jflinchbaugh**
Update new text, icons, etc in the operator catalog.

JIRA: https://issues.sonatype.org/browse/INT-2542

Screen Shot 2020-04-20 at 7 30 44 PM
Screen Shot 2020-04-20 at 7 31 06 PM
Screen Shot 2020-04-20 at 7 31 14 PM

### CODE REVIEW

File: deploy/olm-catalog/nxrm-operator-certified/3.22.0/nxrm-operator-certified.v3.22.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -136,15 +136,85 @@ spec:
...
+    Once the server instance is created by the operator and running,
+    you'll want to expose the service as you see fit:
+    1. Create the service: `kubectl expose deployment --type=NodePort <full name of the new pod>`
+    2. Create a Route to that service for nexus.port (8081).
+
+    By default, the Nexus Repository starts up in OSS mode until a license is installed.
+
+    The Nexus Repository can be further configured via the NexusRepo custom resource definition:
+
+    | Parameter                                   | Description                         | Default                                 |
```
Comment (jflinchbaugh) [inline deploy/olm-catalog/nxrm-operator-certified/3.22.0/nxrm-operator-certified.v3.22.0.clusterserviceversion.yaml:157]: table of configs was brought over from the readme of the helm chart.

File: deploy/olm-catalog/nxrm-operator-certified/3.22.0/nxrm-operator-certified.v3.22.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -1,279 +0,0 @@
...
-      ]
-    capabilities: Basic Install
-    categories: "Integration & Delivery"
-    description: Sonatype Nexus is an open source repository manager
-    containerImage: registry.connect.redhat.com/sonatype/nxrm-operator-certified
-    repository: https://github.com/sonatype/operator-nxrm3
-    createdAt: 2020-04-07
-    support: Sonatype
-    certified: "true"
-  name: nxrm-operator-certified.v3.22.0
```
Comment (jflinchbaugh) [inline deploy/olm-catalog/nxrm-operator-certified/3.22.0/nxrm-operator-certified.v3.22.0.clusterserviceversion.yaml:133]: I was told that following the app version in the operator file names and versions is a fine approach, but it sure does cause a lot of file churn. yuck. I might change my mind on this at some point.

File: bundle/nxrm-operator-certified.crd.yaml [hunk 3]
```diff
@@ -1,14 +1,14 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: nxrms.apm.nxrm.com
+  name: nexusrepos.apm.nxrm.com
```
Comment (adrianpowell) [inline bundle/nxrm-operator-certified.crd.yaml:4]: what is the significance of `apm.nxrm.com`? Should we be trying to remove the `nxrm.com` as well?

File: deploy/olm-catalog/nxrm-operator-certified/3.22.1/nxrm-operator-certified.v3.22.1.clusterserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+            "statefulset": {
+              "enabled": false
+            },
+            "tolerations": []
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: "Integration & Delivery"
+    description: Nexus Repository is the central source of control for all binaries and build artifacts across your DevOps pipelines.
```
Comment (adrianpowell) [inline deploy/olm-catalog/nxrm-operator-certified/3.22.1/nxrm-operator-certified.v3.22.1.clusterserviceversion.yaml:127]: minor: yaml has a few mechanisms for wrapping long lines. See https://www.baeldung.com/yaml-multi-line

File: bundle/nxrm-operator-certified.crd.yaml (hunk 3 above)
Comment (jflinchbaugh) [reply bundle/nxrm-operator-certified.crd.yaml:4]: It seemed relatively inconsequential, so I haven't tried to redefine that yet. I'd like to save that experiment for another time. The changes I made here focused around the presentation of the operator on the hub.

File: deploy/olm-catalog/nxrm-operator-certified/3.22.1/nxrm-operator-certified.v3.22.1.clusterserviceversion.yaml (hunk 4 above)
Comment (jflinchbaugh) [reply deploy/olm-catalog/nxrm-operator-certified/3.22.1/nxrm-operator-certified.v3.22.1.clusterserviceversion.yaml:127]: Good suggestion, I'll see about wrapping this better.

**Comment by jflinchbaugh** [comment]
> The change looks good. Couple minor comments which you can take or not. (I didn't validate the base-64 encoding of the image, trusting you on that :) )

The screenshot shows the icon that Kevin suggested I use. :)

**Pull request merged**